src/utils/jwt.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable require-jsdoc */
```

Comment:
Add JSDoc

Reply:
For this, @b0nbon1 has documented the file. I left this for his PR to be merged